src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -889,6 +863,17 @@ def begin_string(self):
 
 
 def _sig_set_to_string(sig_set):
+    '''
+    Converts the provided set of signatures in a lexically sorted
```

Comment:
s/in a/into a/ perhaps?

Reply:
Yes, definitely. It's one of those where your brain reads what it wants, not what's there. :)